### PR TITLE
enable trigger for release repo

### PR DESCRIPTION
**Pull request by BenTheElder**
TODO(followup): looks like the release repo also needs OWNERS if we want to add approve / tide in the future

### CODE REVIEW

**Comment by BenTheElder** [comment]
required to start testing https://github.com/kubernetes/test-infra/pull/7658

**Comment by BenTheElder** [comment]
/retest
:/

File: prow/plugins.yaml [hunk 1]
```diff
@@ -274,6 +274,9 @@ plugins:
   - approve
   - blunderbuss
 
+  kubernetes/repo-infra:
```
Comment (krzyzacy) [inline prow/plugins.yaml]: k/release?

**Pull request merged**